app/server/lib/DocApi.ts

```diff
@@ -919,7 +953,8 @@ export class DocWorkerApi {
     );
 
     /**
-     @deprecated please call to POST /webhooks instead, this endpoint is only for sake of backward compatibility
```

Comment:
Some autoformatting must have kicked in here and reformatted these comments to be < 100 characters in line length. Let me know if they should be reverted back.

Reply:
That's fine.